.github/workflows/docker-moby-latest.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests agains Latest Docker Moby
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
I think we don't need it on individual PRs.

Reply:
Indeed, that's why it's part of the revert commit 😅